components/dashboard/src/service/metrics.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { GitpodHostUrl } from "@gitpod/gitpod-protocol/lib/util/gitpod-host-url";
+import { MetricsReporter } from "@gitpod/public-api/lib/metrics";
+import { getExperimentsClient } from "../experiments/client";
+
+const originalConsoleError = console.error;
+
+const options = {
+    gitpodUrl: new GitpodHostUrl(window.location.href).withoutWorkspacePrefix().toString(),
+    clientName: "dashboard",
+    clientVersion: "",
+    logError: originalConsoleError.bind(console),
+    isEnabled: () => getExperimentsClient().getValueAsync("dashboard_metrics_enabled", false, {}),
```

Comment:
@akosyakov We could default to `true` here I think, to favor "more data" over "be totally safe against potential issues".

Reply:
Ah, did not think about Dedicated. Yeah, I guess then false is better.